### ISSUE TITLE
Beta thalassemia major

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6065]
**Preferred term label:**
Beta thalassemia major

**Synonyms**
Cooley's anemia, Mediterranean anemia

**Definition (free text, please give PubMed ID)**
Hereditary blood disorder characterized by severe anemia within the first two years of life. Findings in untreated individuals are growth retardation, pallor, jaundice, poor musculature, hepatosplenomegaly, leg ulcers, development of masses from extramedullary hematopoiesis and skeletal changes that result from expansion of the bone marrow.
PMID: 20492708 

**Parent term (use hpo.jax.org/app)**
Thalassemia

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 613985 

**Your nano-attribution (ORCID)**
HBB (OMIM: 141900), LCRB (OMIM: 152424)

**Comment by pnrobinson** [comment]
This is a disease and not a phenotype. Here is the phenotype that is most closely related to the disease
https://hpo.jax.org/app/browse/term/HP:0011906
